sequence_check_digit/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
If you finally decide to change the license to LGPL-3, then remember that you need to change this image and the header of almost every file.